### ISSUE TITLE
Unable to install Gryphon

### ISSUE THREAD

**Issue by MillerZ51** [vinivendra/homebrew-gryphon#2]
**Describe the bug**
Couldn't install Gryphon, tried with homebrew, mint, and cloning git.

**How to reproduce it**
https://vinivendra.github.io/Gryphon/installingGryphon.html
I've tried everything here with my Mac.. (Homebrew, Mint and Cloning)

**The error log: (Homebrew)**
```
username@Omers-MacBook-Pro ~ % brew install vinivendra/gryphon/gryphon
==> Installing gryphon from vinivendra/gryphon
==> Downloading https://rubygems.org/downloads/xcodeproj-1.16.0.gem
Already downloaded: /Users/username/Library/Caches/Homebrew/downloads/9e7f02841e532e6e0113019acaa6459ebb34d50c4f4f856356ef4d38cfa5e2d4--xcodeproj-1.16.0.gem
==> Downloading https://github.com/vinivendra/Gryphon/archive/v0.13.1.tar.gz
Already downloaded: /Users/username/Library/Caches/Homebrew/downloads/e953e549d883b8116e8ef36258486916b75b4e422f6debf5c1e9364b17c279c7--Gryphon-0.13.1.tar.gz
==> Downloading https://rubygems.org/downloads/xcodeproj-1.16.0.gem
Already downloaded: /Users/username/Library/Caches/Homebrew/downloads/9e7f02841e532e6e0113019acaa6459ebb34d50c4f4f856356ef4d38cfa5e2d4--xcodeproj-1.16.0.gem
==> gem install --install-dir /usr/local/Cellar/gryphon/0.13.1/libexec/vendor /U
==> swift build --disable-sandbox
Last 15 lines from /Users/username/Library/Logs/Homebrew/gryphon/02.swift:
2021-01-19 13:37:58 +0200

swift
build
--disable-sandbox

error: terminated(72): HOMEBREW_CACHE=/Users/username/Library/Caches/Homebrew HOMEBREW_CORE_DEFAULT_GIT_REMOTE=https://github.com/Homebrew/homebrew-core HOMEBREW_PREFIX=/usr/local HOMEBREW_SDKROOT=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk HOMEBREW_LIBRARY_PATHS=/usr/local/opt/openssl@1.1/lib:/usr/local/opt/readline/lib:/usr/local/opt/ruby/lib:/usr/local/lib:/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries HOMEBREW_USER_AGENT=Homebrew/2.7.5 (Macintosh; Intel Mac OS X 11.1) HOMEBREW_DEFAULT_TEMP=/private/tmp CMAKE_PREFIX_PATH=/usr/local/opt/openssl@1.1:/usr/local/opt/readline:/usr/local/opt/ruby:/usr/local/opt/openjdk:/usr/local HOMEBREW_USER_AGENT_CURL=Homebrew/2.7.5 (Macintosh; Intel Mac OS X 11.1) curl/7.64.1 HOMEBREW_DEFAULT_LOGS=/Users/username/Library/Logs/Homebrew HOMEBREW_OPT=/usr/local/opt CPATH=/usr/local/include HOMEBREW_BOTTLE_DOMAIN=https://homebrew.bintray.com CC=clang HOMEBREW_REQUIRED_RUBY_VERSION=2.6.3 OBJCXX=clang++ PKG_CONFIG_PATH=/usr/local/opt/libyaml/lib/pkgconfig:/usr/local/opt/openssl@1.1/lib/pkgconfig:/usr/local/opt/readline/lib/pkgconfig:/usr/local/opt/ruby/lib/pkgconfig HOMEBREW_FORMULA_PREFIX=/usr/local/Cellar/gryphon/0.13.1 PATH=/usr/local/Homebrew/Library/Homebrew/shims/mac/super:/usr/local/opt/openssl@1.1/bin:/usr/local/opt/ruby/bin:/usr/local/opt/openjdk/bin:/usr/local/opt/kotlin/bin:/usr/bin:/bin:/usr/sbin:/sbin HOMEBREW_MACOS_VERSION_NUMERIC=110100 HOMEBREW_ENV=super TERM=xterm-256color HOMEBREW_CURL=curl HOMEBREW_PRODUCT=Homebrew HOMEBREW_CELLAR=/usr/local/Cellar SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.nXFjzkAgJU/Listeners HOMEBREW_MACOS_SYSTEM_RUBY_NEW_ENOUGH=1 HOMEBREW_OPTIMIZATION_LEVEL=Os USER=username HOMEBREW_LOGS=/Users/username/Library/Logs/Homebrew HOMEBREW_CXX=clang++ HOMEBREW_ISYSTEM_PATHS=/usr/local/include:/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers LC_ALL=en_US.UTF-8 HOMEBREW_OPTFLAGS=-march=nehalem LIBRARY_PATH=/usr/local/lib HOMEBREW_ERROR_PIPE=/private/tmp/homebrew20210119-9800-gj5uhx/socket HOMEBREW_OS_VERSION=macOS 11 HOMEBREW_COMMAND_DEPTH=2 HOMEBREW_GIT=git HOMEBREW_ARCHFLAGS= PKG_CONFIG_LIBDIR=/usr/lib/pkgconfig:/usr/local/Homebrew/Library/Homebrew/os/mac/pkgconfig/11 CURL_HOME=/Users/username HOMEBREW_ANALYTICS_ID=UA-76679469-1 HOMEBREW_VERSION=2.7.5 OBJC=clang TMP=/private/tmp CMAKE_LIBRARY_PATH=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries SHELL=/bin/zsh HOMEBREW_DEPENDENCIES=libyaml,openssl@1.1,readline,ruby,openjdk,kotlin HOMEBREW_BOTTLE_DEFAULT_DOMAIN=https://homebrew.bintray.com HOMEBREW_SYSTEM=Macintosh HOMEBREW_LIBRARY=/usr/local/Homebrew/Library HOMEBREW_BREW_FILE=/usr/local/bin/brew HOMEBREW_GIT_CONFIG_FILE=/usr/local/Homebrew/.git/config HOMEBREW_REPOSITORY=/usr/local/Homebrew CARGO_HOME=/Users/omermiller/Library/Caches/Homebrew/cargo_cache LOGNAME=omermiller HOMEBREW_DEVELOPER_DIR=/Library/Developer/CommandLineTools HOMEBREW_MACOS_VERSION=11.1 HOMEBREW_CORE_GIT_REMOTE=https://github.com/Homebrew/homebrew-core HOMEBREW_BREW_GIT_REMOTE=https://github.com/Homebrew/brew HOMEBREW_MINIMUM_GIT_VERSION=2.14.3 HOMEBREW_ANALYTICS_USER_UUID=1C1C01B4-2922-4788-A5A3-270552B096A3 HOMEBREW_INCLUDE_PATHS=/usr/local/opt/openssl@1.1/include:/usr/local/opt/readline/include:/usr/local/opt/ruby/include:/usr/local/opt/openjdk/include HOMEBREW_CC=clang HOMEBREW_RUBY_PATH=/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby MAKEFLAGS=-j8 HOME=/private/tmp/gryphon-20210119-10471-1pfq3dh/Gryphon-0.13.1/.brew_home _JAVA_OPTIONS=-Duser.home=/Users/username/Library/Caches/Homebrew/java_cache HOMEBREW_RUBY_WARNINGS=-W0 GOPATH=/Users/username/Library/Caches/Homebrew/go_mod_cache PWD=/Users/username HOMEBREW_DEFAULT_CACHE=/Users/username/Library/Caches/Homebrew HOMEBREW_AUTO_UPDATE_CHECKED=1 SHLVL=0 HOMEBREW_PREFER_CLT_PROXIES=1 GOCACHE=/Users/username/Library/Caches/Homebrew/go_cache CMAKE_INCLUDE_PATH=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers HOMEBREW_PROCESSOR=Intel TEMP=/private/tmp __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x0 TMPDIR=/private/tmp HOMEBREW_BREW_DEFAULT_GIT_REMOTE=https://github.com/Homebrew/brew CXX=clang++ HOMEBREW_TEMP=/private/tmp HOMEBREW_CCCFG=sa HOMEBREW_CC_LOG_PATH=/Users/username/Library/Logs/Homebrew/gryphon/02.swift SDKROOT=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk /usr/bin/xcrun --sdk macosx --find xctest output:
    xcrun: error: unable to find utility "xctest", not a developer tool or in PATH
```

    

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/vinivendra/homebrew-gryphon/issues

Maybe I did something wrong, but I'm pretty sure I just did what the guide told me to..
Seems that each way I've tried to install Gryphon, there is a problem with the "xctest" utility..

**Your environment (please complete the following information):**
OS: macOS 11.1 (Big Sur)
Xcode: 12.3
Gryphon version: 0.13.1

**Comment by vinivendra** [comment]
Hello! Coincidentally, I had the same issue just the other day. If I recall correctly, it was a problem with Xcode's command line tools. Could you try this for me and tell me if it works?

- Open Xcode
- Open the preferences (**⌘+,**)
- Go to **Locations** on the top bar
- Click on the **Command Line Tools** drop-down menu and select any option from the list
- Try installing Gryphon again

**Comment by MillerZ51** [comment]
Hello vinivendra!
Thanks for the really fast responding. sorry for taking time responding myself.

It works now.
I did install again and now I think Gryphon is working..
Sadly I am having a problem with my Android Studio on my Mac, so I can't confirm that everything is fine just yet.
But I've succeeded to create some files with the commands:
`gryphon init myAwesomeiOSApp.xcodeproj/`
and
`gryphon init myAwesomeiOSApp.xcodeproj/ --target=<target name>`
from your tutorial. So I think it's OK :)
I'll fix Android Studio and let you know If I have any problem..
Thanks again, it seems like you've done an amazing job here!
